contracts/src/core/libraries/v3/config-registry/ConfigRegistryTypes.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+library ConfigRegistryTypes {
+    struct NameSet {
+        mapping(bytes32 => string) names;
+        string[] nameList;
+    }
+
+    struct Bytes32Cfg {
+        mapping(bytes32 => bytes32) values;
+        mapping(bytes32 => string) extraInfo;
```

Comment:
curious, why not store them as as single mapping to a struct{values, extraInfo} and have a single getter that returns both?

Reply:
In this particular case, I don't think structifying this makes the code any clearer, and bundling it doesn't really serve any on-chain use case. extraInfo is not meant to be consumed on-chain at all, and will typically just be a wasted SLOAD if it is bundled into a getter.

Reply:
I was assuming the compiler would be smart enough to only load the part of the struct that is actually used, but maybe not.